ark-examples/file.go

```diff
@@ -33,7 +34,19 @@ type FileObjectStore struct {
 
 func (f *FileObjectStore) Init(config map[string]string) error {
 	f.log.Infof("FileObjectStore.Init called")
-	return os.MkdirAll(getRoot(), 0755)
+
+	bucket, ok := config["bucket"]
+	if !ok {
```

Comment:
Is this being done to be super explicit? Otherwise you could just pass `config["bucket"]`. Same question below.

Reply:
Yes. I had forgotten about the default values for missing keys, so I'll update that.